db/migrate/20151118235203_remove_default_value_from_items_brand.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveDefaultValueFromItemsBrand < ActiveRecord::Migration
+  def change
+    change_column_default(:items, :brand, 'nil')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.